admin/workspace-providers/index.md

```diff
@@ -22,8 +22,6 @@ specifies the Kubernetes cluster containing the Coder deployment. This allows
 users to create workspaces in the same cluster as the Coder deployment with no
 additional configuration.
 
-You cannot delete the `built-in` workspace provider.
```

Comment:
You can. I tested with a self-hosted as well as in SaaS. You just need to remove the default organization

Reply:
I'd leave this out. We will have a no-provider experience soon.

Reply:
Do you mean I should keep line 25? This PR removes it.

Reply:
Remove the line.